src/components/addCategoryModal/AddCategoryModal.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function AddCategoryModal() {
+    return(
+        <div>
+            <h1>Add Category Modal working</h1>
+        </div>
+    )
+}
+
+export default AddCategoryModal;
```

Comment:
@natalievasquez11 Really good job!  I forgot to mention that it is standard to end the JS files with a newline so there is an empty line as the last line.  Very small change, but it is described in more detail here: https://evanhahn.com/newline-necessary-at-the-end-of-javascript-files/